core/config/initializers/assets.rb

```diff
@@ -1,6 +1,27 @@
+# These precompile sets are split for readablity
+# scripts
+ Rails.application.config.assets.precompile += %w(
```

Comment:
This is expected to be in `initializer` in Rails engine IMHO.

see https://github.com/activeadmin/activeadmin/blob/c6347ea2429d0f052e167ba472a35f1d1061124e/lib/active_admin/engine.rb#L3-L10 for example what I mean

@parndt what do you think?